modules/reference/pages/config/server-configuration-overview.adoc

```diff
@@ -194,6 +195,29 @@ For example, if you enter `${my.env.var}` in the `server.xml` file, it looks for
 . my_env_var
 . MY_ENV_VAR
 
+Variables can be loaded from files located in directories that are specified by the `VARIABLE_SOURCE_DIRS` environment variable.
+The default location is `${server.config.dir}/variables`.
+The name of the file is used as the variable name, and the contents of the file are used as the variable value, as shown in the following example:
+----
```

Comment:
dont think this belongs in a code block. Just rewrite it as plain text- but should clarify what is meant by "content of the file". As Brett mentioned in the comment, the content of the file is just a text string. So something like:


The name of the file is used as the variable name, and the contents of the file are used as the variable value. For example, you can create a file in the `variables` directory that is named `httpPort` with the text string `9080` as the only content. You can then use the variable `${httpPort}` in the `server.xml` file and it resolves to `9080`.

If you create a file in a directory within the `variables` directory, then the parent directory is added to the variable name. For example, if you create a `${httpPort}` file in a `ports` subdirectory within the `variables` directory, the variable name is `${ports/httpPort}`.